### PR TITLE
fix(deps): remove indirect `tokio-tungstenite` dependencies

### PR DESCRIPTION
**Pull request by hanabi1224**
## Summary of changes

As part of https://github.com/ChainSafe/forest/issues/4012

Dropping the direct `tokio-tungstenite` usage will come after https://github.com/ChainSafe/forest/pull/4010

Changes introduced in this pull request:

- remove indirect `tokio-tungstenite` dependencies by dropping `ws` feature from `axum`
- test websocket rpc endpoints in `Offline RPC tests`

## Reference issue to close (if applicable)

Closes

## Change checklist

- [x] I have performed a self-review of my own code,
- [x] I have made corresponding changes to the documentation,
- [x] I have added tests that prove my fix is effective or that my feature works (if possible),
- [x] I have made sure the [CHANGELOG][1] is up-to-date. All user-facing changes should be reflected in this document.

[1]: https://github.com/ChainSafe/forest/blob/main/CHANGELOG.md

### CODE REVIEW

File: scripts/tests/calibnet_offline_rpc_check.sh [hunk 1]
```diff
@@ -43,5 +43,15 @@ if [ $exit_code -ne 0 ]; then
     exit 1
 fi
 
+# Compare the ws endpoints
+$FOREST_TOOL_PATH api compare "$snapshot" --forest /ip4/127.0.0.1/tcp/8080/http --lotus /ip4/127.0.0.1/tcp/8081/http --n-tipsets 5 --ws
```
Comment (elmattic) [inline scripts/tests/calibnet_offline_rpc_check.sh:47]: This works?
If we use the `--ws` flag we should pass `/ip4/127.0.0.1/tcp/8080/ws`

**Review by elmattic — CHANGES_REQUESTED** [review]

File: scripts/tests/calibnet_offline_rpc_check.sh (hunk 1 above)
Comment (elmattic) [reply scripts/tests/calibnet_offline_rpc_check.sh]: Interesting.

Comment (hanabi1224) [reply scripts/tests/calibnet_offline_rpc_check.sh]: I'll investigate the issue separately, fixed the PR

**Review by elmattic — APPROVED** [review]

File: scripts/tests/calibnet_offline_rpc_check.sh (hunk 1 above)
Comment (elmattic) [reply scripts/tests/calibnet_offline_rpc_check.sh]: Sure.

Comment (elmattic) [reply scripts/tests/calibnet_offline_rpc_check.sh]: Our error handling is misguiding us here; we won't exercise web socket communication mode if we don't use `/ws` in urls when using the `--ws` flag.

I think we should validate things upfront.

Comment (hanabi1224) [reply scripts/tests/calibnet_offline_rpc_check.sh]: > I think we should validate things upfront.

@elmattic That makes sense, could you open an issue for that?

Comment (elmattic) [reply scripts/tests/calibnet_offline_rpc_check.sh]: I have a fix here: https://github.com/ChainSafe/forest/pull/4018

**Pull request closed without merging**